spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -237,6 +237,8 @@ class CodeEditor(TextEditBaseWidget):
     get_completions = Signal(bool)
     go_to_definition = Signal(str, int, int)
     sig_show_object_info = Signal(int)
+    # str: object name , str: object signature/documentation,

```

Comment:
There's no need for this comment @dalthviz. @goanpeca only used as an example of how documentation for signals should look like. So please remove this line and leave a blank space.

Also move the comment below and the signal (`sig_show_completion_object_info`) to be below `sig_go_to_uri`.

Reply:
Ok, I understand better now the meaning of this comment. Let's change it a bit like this:

# Description of the signal
# Description of the parameters of the signal, one per line.

Reply:
And also, please move this below `sig_go_to_uri`.